docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,155 @@
+JDBC Poller Input Operator
+=============
+
+## Operator Objective
+This operator scans jdbc table in parallel fashion. The jdbc table is table in jdbc complaint database. To provide parallelism during read each instance of Apex input operator also known as operator partition reads one portion of jdbc table. Based on requested operator instances number of non-polling static partitions are created to scan table portitions. The non-polling partition will scan the table portion only once. Additional single operator partition is created to scan newly added data incrementally. 
+
+#### Assumption
+Assumption is that there is an ordered column using which range queries can be formed. That means database has a column which has unique constraint as well as the newly inserted record should have column value more than max value in that column.
+
+## Use cases
+1. Scan huge database tables to either copy to other database or process it using **Apache Apex**. The example application to copy database contents to HDFS is available in [examples repository](https://github.com/DataTorrent/examples/tree/master/tutorials/jdbcIngest)
+
+## How to Use?
+The tuple type in the abstract class is a generic parameter. Concrete subclasses need to choose an appropriate class (such as String or java class for [POJO](https://en.wikipedia.org/wiki/Plain_Old_Java_Object) etc for the generic parameter) and also implement a couple of abstract methods: getTuple(ResultSet) to parse database row to chosen class and emitTuple(T) to emit the tuple.
+
+In principle, no ports need be defined in the rare case that the operator simply writes tuples to some external sink or merely maintains aggregated statistics. But in most common scenarios, the tuples need to be sent to one or more downstream operators for additional processing such as parsing, enrichment or aggregation; in such cases, appropriate output ports are defined and the emitTuple(T) implementation dispatches tuples to the desired output ports.
+
+Couple of concrete implementations are provided in Malhar:
+
+1. [JdbcPOJOPollInputOperator](https://github.com/apache/apex-malhar/blob/master/library/src/main/java/com/datatorrent/lib/db/jdbc/JdbcPOJOPollInputOperator.java): It uses java Object for the generic parameter. You need to set output port attribute TUPLE_CLASS to define your POJO class name to define Object type. The record fetched from database table will be parsed as object of configured POJO class. This operator defines a single output port and processes each database table record one by one as a tuple object. Details are discussed below.
```

Comment:
Suggest moving the sentence "This operator defines a single ...." to before this one: "You need to set output..."

"set output port" => "set the output port"
"from database table" => "from the database table"
"parsed as object of configured" => "parsed, using the `getTuple` method mentioned above, as an object of the configured"

Here, since POJO is part of the operator name, it seems reasonable to provide an expansion and a link.